### DISCUSSION TITLE
Detection speed of connected clients

### DISCUSSION THREAD

**Discussion by TheQue42** [goofball222/unifi#92] [General]
Hi, 

I am using the unifi-controller together with home-assistant, for presence detection. Lately though (yes, unfortunately somewhat unspecific) I have noticed that unifi (sometimes, often) seems to be really slow in detecting that a client has connected/disconnected. (especially disconnection)

I've been sitter with a timer and checking the logs, and it takes about five minutes before I can find the 
```
[2021-07-07T08:14:10,753] XXX <stamgr> INFO  event  - [event] User[aa:bb:cc:dd:ee:ff] disconnected from "WIFI_NET_X" (1m 42s connected, 20.34K bytes, last AP[aa:bb:cc:dd:ee:ff])
[2021-07-07T08:22:12,841] XXX <inform_stat-1> INFO  event  - [event] User[aa:bb:cc:dd:ee:ff] has connected to AP[78:8a:20:48:76:c7] with SSID "WIFI_NET_X" on "channel 44(na)"
```
Is this really normal? I **think** it was way faster before, at least connection-detection....

**Comment by goofball222** [comment]
There shouldn't be anything in the container itself that would cause that since it's just a bundle of the minimum software setup required along with the UI distributed UniFi package. Since the messages / updates are getting there eventually, all of the ports, etc. are open. Could very well be something with the UniFi software or firmware or a combination thereof causing it, I've never really paid attention to how long it takes for clients to appear/disappear in the interface. 

Would be worth checking / posting on the UI community to see if anyone else has reported it: https://community.ui.com/questions

**Reply by TheQue42** [reply]
Of course. I just didnt know if you had any more detailed knowledge about changes in the packages from Unifi that may cause these things. I'll check the forums, just as soon as I can get my thumb out :-)
